docs/content.zh/docs/dev/table/materialized-table/statements.md

```diff
@@ -200,11 +200,11 @@ CREATE MATERIALIZED TABLE my_materialized_table
 CREATE MATERIALIZED TABLE my_materialized_table_continuous
     PARTITIONED BY (ds)
     WITH (
-        'format' = 'json',
```

Comment:
Can you also help fix the line 168 & line 169,  line 175 & line 176, correct the REFRESH_MODE and FRESHNESS position, the FRESHNESS should be above the REFRESH_MODE.

Reply:
@lsyldliu done.